playbooks/webapp/tasks/delete.yaml

```diff
@@ -1,10 +1,13 @@
 ---
```

Comment:
I'm running into a dependency error at the `Delete security groups` step of this playbook: `An error occurred (DependencyViolation) when calling the DeleteSecurityGroup operation: resource sg-0967dbe0b86f1d643 has a dependent object`. I haven't been able to identify the root cause yet but wanted to go ahead and finish this review so...I'll keep looking into it but I'm thinking there may be some other resources that need to be deleted before that step in the high availability scenario?